monitor/cli.py

```diff
@@ -145,5 +179,87 @@ def report(ctx, config_dir, profiles_dir, update_dbt_package, profile_target):
         raise
 
 
+@monitor.command()
+@click.option(
+    '--config-dir', '-c',
+    type=str,
+    default=os.path.join(expanduser('~'), '.edr'),
+    help="Global settings for edr are configured in a config.yml file in this directory "
+         "(if your config dir is HOME_DIR/.edr, no need to provide this parameter as we use it as default)."
+)
+@click.option(
+    '--profiles-dir', '-p',
+    type=str,
+    default=os.path.join(expanduser('~'), '.dbt'),
+    help="Specify your profiles dir where a profiles.yml is located, this could be a dbt profiles dir "
+         "(if your profiles dir is HOME_DIR/.dbt, no need to provide this parameter as we use it as default).",
+)
+@click.option(
+    '--update-dbt-package', '-u',
+    type=bool,
+    default=False,
+    help="Force downloading the latest version of the edr internal dbt package (usually this is not needed, "
+         "see documentation to learn more)."
+)
+@click.option(
+    '--slack-webhook', '-s',
+    type=str,
+    default=None,
+    help="A slack webhook URL for sending alerts to a specific channel (also could be configured once in config.yml)"
+)
+@click.option(
+    '--slack-token', '-st',
+    type=str,
+    default=None,
+    help="A slack token for sending alerts over slack (also could be configured once in config.yml)",
+    cls=RequiredIf,
+    required_if="slack_channel_name"
+)
+@click.option(
+    '--slack-channel-name', '-ch',
+    type=str,
+    default=None,
+    help="The slack channel which all alerts will be sent to (also could be configured once in config.yml)",
+    cls=RequiredIf,
+    required_if="slack_token"
+)
+@click.option(
+    '--profile-target', '-t',
+    type=str,
+    default=None,
+    help="if you have multiple targets for Elementary, optionally use this flag to choose a specific target"
+)
+@click.pass_context
+def send_report(
+    ctx,
+    config_dir,
+    profiles_dir,
+    update_dbt_package,
+    slack_webhook,
+    slack_token,
+    slack_channel_name,
+    profile_target
+):
+    config = Config(config_dir, profiles_dir, profile_target)
+    anonymous_tracking = AnonymousTracking(config)
+    track_cli_start(anonymous_tracking, 'monitor-send-report', get_cli_properties(), ctx.command.name)
+    try:
+        data_monitoring = DataMonitoring(
+            config=config,
+            force_update_dbt_package=update_dbt_package,
+            slack_webhook=slack_webhook,
+            slack_token=slack_token,
+            slack_channel_name=slack_channel_name
+        )
+        generated_report_successfully = data_monitoring.generate_report()
```

Comment:
cosmetics - send report can call generate_report internally

Reply:
Decided to separate between sending and generating